frontend/.env.local.example

```diff
@@ -6,4 +6,5 @@ NEXT_PUBLIC_FIREBASE_PROJECT_ID=
 NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=
 NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
 NEXT_PUBLIC_FIREBASE_APP_ID=
-NEXT_PUBLIC_GOOGLE_CLIENT_ID=
\ No newline at end of file
+NEXT_PUBLIC_GOOGLE_CLIENT_ID=
+NEXT_PUBLIC_PEERPREP_INNKEEPER_SOCKET_URL=
```

Comment:
You need to add the new env variable to the CI files as well as the github secrets

Reply:
Thanks for the catch: [Update env for github workflows](https://github.com/PeerPrep/peerprep/pull/53/commits/39da179f508d34ad6befceada5f9d8f323076885)